prow/oss/cluster/cluster.yaml

```diff
@@ -90,16 +90,6 @@ spec:
             name: hook
             port:
               number: 8888
-  - host: oss-prow-private.knative.dev
```

Comment:
We can also remove the managed certs and associated ingress annotations too.

Reply:
good point, done, both